hass_nabucasa/remote.py

```diff
@@ -70,6 +71,15 @@ class Certificate:
     fingerprint = attr.ib(type=str)
 
 
+class CertificateStatus(str, Enum):
+    """Representation of the remote UI status."""
+
+    ERROR = "error"
+    GENERATING = "generating"
+    LOADED = "loaded"
```

Comment:
Should we make this `READY` ?

Reply:
No, but I can add a final ready status where loaded is use now, the certificate is loaded before in 1 case.